### PR TITLE
Consolidate wallet logic in integration tests

### PR DESCRIPTION
**Pull request by tudor-malene**
### Why is this change needed?

The bridge logic has to support multiple erc20 tokens mapped to wrapped tokens.
This requires to juggle multiple wallets.
The current wallet logic was scattered around, and hard to follow and change.

### What changes were made as part of this PR:

- refactoring PR 
- consolidate all the wallet logic to the SimWallets type
- minor change to the evm_facade to move the hardcoded "bridge" addresses as they have to be removed

### What are the key areas to look at
- the SimWallets
- the readability of the "simulation_*" tests

### CODE REVIEW

File: integration/simulation/params/wallet_utils.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	"math/big"
+
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/enclave/evm"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/wallet"
+	"github.com/obscuronet/obscuro-playground/integration"
+	"github.com/obscuronet/obscuro-playground/integration/datagenerator"
+)
+
+type SimWallets struct {
+	MCOwnerWallet        wallet.Wallet
```
Comment (joeldudley) [inline integration/simulation/params/wallet_utils.go:13]: @tudor-malene Can this have a better name or a comment? Making me think of a big mac.

File: tools/networkmanager/inject_txs.go [hunk 2]
```diff
@@ -44,7 +46,9 @@ func InjectTransactions(nmConfig Config) {
 		1*time.Second,
 		stats.NewStats(1),
 		[]ethclient.EthClient{l1Client},
-		[]wallet.Wallet{l1Wallet},
+		&params.SimWallets{
+			// todo
```
Comment (joeldudley) [inline tools/networkmanager/inject_txs.go:50]: @tudor-malene Can you expand on this todo?

**Review by joeldudley — APPROVED** [review]
Looks fine, just some small comments

File: tools/networkmanager/inject_txs.go (hunk 2 above)
Comment (tudor-malene) [reply tools/networkmanager/inject_txs.go:50]: ah yeah, that was actually a question to you. let's chat later about this
